ts/features/bonus/cgn/store/actions/otp.ts

```diff
@@ -0,0 +1,14 @@
+import { ActionType, createAsyncAction } from "typesafe-actions";
+import { NetworkError } from "../../../../../utils/errors";
+import { Otp } from "../../../../../../definitions/cgn/Otp";
+
+/**
+ * handle CGN Otp generation
+ */
+export const cngGenerateOtp = createAsyncAction(
```

Comment:
```suggestion
export const cgnGenerateOtp = createAsyncAction(
```

Reply:
thanks!
changed in https://github.com/pagopa/io-app/pull/2882/commits/0c7f2ce791d125d60b89fc83901c40150c741946